Makefile

```
@@ -33,7 +34,14 @@ release-std:
 # for all windows, linux and mac, 64-bit only,
 # using the standard Golang toolchain.
 xc:
-	./release.sh
+	docker build -t rivinebuilder .
+	docker run --rm -v $(shell pwd):/go/src/github.com/rivine/rivine rivinebuilder
+
+# Release images builds and packages release binaries, and uses the linux based binary to create a minimal docker
+release-images: xc
+	docker build -t rivine/rivine:$(version) -f DockerfileMinimal --build-arg binaries_location=release/rivine-$(version)-linux-amd64 .
+	docker push rivine/rivine:$(version)
+	curl -b "active-user=rivine; caddyoauth=$(HUB_JWT)" -X POST --data "image=rivine/rivine:$(version)" "https://hub.gig.tech/api/flist/me/docker"
```

Perhaps we could fail the Make command when `HUB_JWT` is not defined? Might provide better UX than whatever vague error you'll get otherwise.

---

Then again, the JWT might also be outdated, or not contain the `user:memberof:rivine` scope. Since releasing is usually done by a developer, I think we can assume at least **some** knowledge on his part, and not require extensive hand-holding as is the case with end users.

When talking about UX, do we really want to check if the env var is set and return an error if its not, to then not validate whether this JWT is valid or not?

---

Fair enough, it's a pretty flaky release pipeline none the less, but for now it's OK, we can come back later on this. Really not that important for now.

I do start to feel it would be useful if IYO provides a command-line client, which would provide the API as commands, as well as provide ssh access (if possible), as the latter would really make it not necessary to provide any credentials at all.